src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -37,6 +37,12 @@ public class Exam extends DomainObject {
     @Column(name = "test_exam")
     private boolean testExam;
 
+    /**
+     * This boolean indicates whether attendance is checked during exam
+     */
+    @Column(name = "exam_with_attendance_check")
+    private boolean examWithAttendanceCheck;
```

Comment:
```suggestion
    private boolean isExamWithAttendanceCheck;
```

Reply:
boolean attributes in the project are defined without "is" prefix, whereas setter methods do have the prefix